packages/@lwc/template-compiler/src/shared/ir.ts

```diff
@@ -4,48 +4,82 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
+import * as parse5 from 'parse5';
+import treeAdapter from 'parse5/lib/tree-adapters/default';
```

Comment:
It feels fragile to import deep files like this from `parse5`. It looks like we only need `treeAdapter.isElementNode`, which [is a one-liner](https://github.com/inikulin/parse5/blob/9c683e154b8bbdc94237580680a934348406d1a2/packages/parse5/lib/tree-adapters/default.js#L206-L208):

```js
exports.isElementNode = function(node) {
    return !!node.tagName;
};
```

Is there a problem with just using this one-liner in our codebase?

Reply:
Parse5 documentation is really fuzzy on how to import the default tree adapter. You are right, let's get this out of the way.